### PR TITLE
Login nav bar

### PR DESCRIPTION
**Pull request by TylerGauntlett**

### CODE REVIEW

File: src/app/components/login/loginStyles.styl [hunk 1]
```diff
@@ -0,0 +1,40 @@
+body {
```
Comment (Tresky) [inline src/app/components/login/loginStyles.styl]: Will you convert this to Stylus syntax and nest whatever can be nested?

File: src/app/components/home/homeStyles.styl [hunk 2]
```diff
@@ -1,2 +1,111 @@
...
+}
+
+footer {
+  padding 50px 0
+  background-color #f8f8f8
+}
+
+p.copyright {
+  margin 15px 0 0
+}
```
Comment (Tresky) [inline src/app/components/home/homeStyles.styl]: Will you convert this one to Stylus as well? The brackets is the only thing.

File: src/app/shared/navBar/navBarDirective.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  .directive('navBar', function ($log) {
+    return {
+      template: require('./navBarView.html'),
+      restrict: 'A',
+      scope: {
+        flag: '='
+      }
+    }
+  }
+)
```
Comment (Tresky) [inline src/app/shared/navBar/navBarDirective.js]: Does this directive have a parameter on it called 'flag'? If not, let's take it out before committing it.

File: src/app/shared/navBar/navBarStyles.styl [hunk 4]
```diff
@@ -0,0 +1,3 @@
+.topnav {
+  font-size 20px
+}
```
Comment (Tresky) [inline src/app/shared/navBar/navBarStyles.styl]: Brackets :P

File: src/public/index.html [hunk 5]
```diff
@@ -8,8 +8,11 @@
...
 
     <base href="/">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/bootstrap/3.3.7/css/bootstrap.min.css">
   </head>
   <body>
+    <div nav-bar></div>
     <div ui-view class="site"></div>
+    <script src="https://cdn.jsdelivr.net/g/jquery@3.1.1,tether@1.4.0,bootstrap@3.3.7"></script>
   </body>
 </html>
```
Comment (Tresky) [inline src/public/index.html]: Did you get this to work without reordering the JS items? If so, that's super awesome!

**Review by Tresky — CHANGES_REQUESTED** [review]

File: src/app/components/login/loginStyles.styl (hunk 1 above)
Comment (TylerGauntlett) [reply src/app/components/login/loginStyles.styl]: Done

File: src/app/components/home/homeStyles.styl (hunk 2 above)
Comment (TylerGauntlett) [reply src/app/components/home/homeStyles.styl]: Done

File: src/app/shared/navBar/navBarDirective.js (hunk 3 above)
Comment (TylerGauntlett) [reply src/app/shared/navBar/navBarDirective.js]: Done

File: src/public/index.html (hunk 5 above)
Comment (TylerGauntlett) [reply src/public/index.html]: Ended up needed jquery first but could remove teather as we are using bootstrap 3 again.

File: src/app/shared/navBar/navBarStyles.styl (hunk 4 above)
Comment (TylerGauntlett) [reply src/app/shared/navBar/navBarStyles.styl]: Done

**Review by Tresky — APPROVED** [review]

**Pull request merged**